guides/common/modules/proc_configuring-dns-dhcp-and-tftp.adoc

```diff
@@ -1,20 +1,11 @@
-[id="configuring-dns-dhcp-and-tftp_{context}"]
-= Configuring DNS, DHCP, and TFTP on {ProductName}
+[id="configuring-dns-dhcp-and-tftp-on-projectserver_{context}"]
+= Configuring DNS, DHCP, and TFTP on {ProjectServer}
```

Comment:
This does not work for "Smart Proxy Servers". So either go with "SmartProxy" or use "ProductName" as before:

```
$ rg ProductName
guides/doc-Installing_Proxy/master.adoc
6::ProductName: {SmartProxyServer}

guides/doc-Quickstart/master.adoc
6::ProductName: {ProjectServer}

guides/doc-Installing_Server/master.adoc
6::ProductName: {ProjectServer}

guides/common/attributes-satellite.adoc
104::ProductName: {ProjectName}

guides/common/assembly_configuring-external-dhcp.adoc
5://Configuring an External DHCP Server to Use with {ProductName}
8://Configuring {ProductName} with an External DHCP Server

guides/doc-Installing_Server_Disconnected/master.adoc
6::ProductName: {ProjectServer}
...
```

Reply:
I guess so. I still don't like the anchor itself, but :shrug: Maybe use "{smart-proxy-context}" in the anchor instead? (title is good!) I'm not sure if it's any better than "productname". Your call.

Reply:
A lot of things about {ProductName} feel weird to me. But it is currently used in master (in the module heading and in its intro), which I think should mean it's okay to use it in the ID as well. In other words, if it's inappropriate for the ID, it should be changed in the title + intro, and then would just start us down the path of another rabbit hole :upside_down_face: 

So let's keep it the way it is now, and if anyone thinks of a better idea for how to handle this module, we can do that in another PR. @maximiliankolb

Reply:
Full ACK!